source/_components/sensor.magicseaweed.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Magicseaweed Sensor"
+description: "How to integrate Magicseaweed within Home Assistant."
+date: 2018-06-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: magicseaweed.png
+ha_category: Sensor
+featured: false
+ha_release: "0.72"
+ha_iot_class: "Cloud Polling"
+---
+
+The 'magicseaweed' platform uses the [Magicseaweed Forecast API](https://magicseaweed.com/developer/forecast-api) as a source for surf forecasting data for the surf spots of your choice.
+
+You need an API key which is free but requires [registration](https://magicseaweed.com/developer/sign-up). Magicseaweed is limiting users of the API to 2 spots for the free plan.
+
+To add Magicseaweed forecasts to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: magicseaweed
+    name: Hurricane Spot
+    api_key: YOUR_API_KEY
+    spot_id: 1092
+    hours:
+      - 6AM
+      - 9AM
+      - 12PM
+      - 3PM
+      - 6PM
+    monitored_conditions:
+      - swell_forecast
+      - min_breaking_swell
+      - max_breaking_swell
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration